tool/tctl/common/auth_command.go

```diff
@@ -748,6 +754,53 @@ func (a *AuthCommand) checkProxyAddr(clusterAPI auth.ClientI) error {
 	return trace.BadParameter("couldn't find registered public proxies, specify --proxy when using --format=%q", identityfile.FormatKubernetes)
 }
 
+func (a *AuthCommand) logUngrantedTTL(clusterAPI auth.ClientI) error {
+	rc := clockwork.NewRealClock()
+	var addr string
+	var u *utils.NetAddr
+	if a.proxyAddr != "" {
+		var err error
+		u, err = utils.ParseAddr(a.proxyAddr)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		addr = u.Addr
+	}
```

Comment:
nit: `u` can stay scoped inside the block below
```suggestion
	if a.proxyAddr != "" {
		u, err := utils.ParseAddr(a.proxyAddr)
		if err != nil {
			return trace.Wrap(err)
		}
		addr = u.Addr
	}
```